### ISSUE TITLE
onfidoSafeWindow causing test failure in Mocha-webpack

### ISSUE THREAD

**Issue by Sandbagger** [onfido/onfido-sdk-ui#969]
## What was the expected behaviour?

Successfully run a suite of tests using Mocha, that includes onfido-sdk-ui in a React Component.

## What happened instead?

The test bombed out with the following error:

```
ReferenceError: onfidoSafeWindow8xmy484y87m239843m20 is not defined
```
Possibly related to Woopra safe window: https://github.com/onfido/onfido-sdk-ui/pull/502

## Version info:
	•	Onfido SDK: "version": "5.8.0"
	•	OS: Catalina
	•	Browser: n/a
	•	Device: MacBook Pro
	•	Node: v8.9.4;
	•	Webpack: v3.10.0
	•	Framework or relevant dependencies (if applicable):
	•	Rails: v5.1.6.1
	•	React v15.6.1

Rails / React project using mocha-webpack as a test runner and webpack precompiler.

## Integration configuration:

```
import React, { Component } from 'react';
import { init } from 'onfido-sdk-ui';
import HttpRequestor from '../..//http_requestor';

export default class OnfidoIdentityUpload extends Component {
  constructor(props) {
    super(props);
    this.state = {
      token: null,
    };
  }

  componentDidMount() {
   // fetch token
  }

  componentDidUpdate(prevProps, prevState) {
    if (this.state.token !== prevState.token) {
      this.launchOnfidoSDK();
    }
  }

  launchOnfidoSDK = () => {
    return (
      this.state.token &&
      init({
       …
      })
    );
  };

  render() {
    return (
      <div id="onfido-mount" className="onfido-upload">
      </div>
    );
  }
}

```
package.json script:
```
"scripts": {
    "test": "bundle exec rake i18n:js:export && NODE_ENV=test node --max_old_space_size=4096 ./node_modules/mocha-webpack/bin/mocha-webpack --opts ./test/mocha-webpack.opts --webpack-config ./config/webpack/test.js \"test/**/*.spec.js\""
}
```

Run mocha-webpack with the following options:

```
--colors
--require babel-core/register
--require test/support/helper
```

Webpack config:

```
const merge = require('webpack-merge')
const sharedConfig = require('./shared.js')

module.exports = merge(sharedConfig, {
  devtool: 'inline-source-map',
  externals: {
    cheerio: 'window',
    'react/addons': 'react',
    'react/lib/ExecutionEnvironment': 'react',
    'react/lib/ReactContext': 'react',
  }
})
```

Steps to reproduce:

```
npm test
```

**Comment by andra-radu** [comment]
Hi @Sandbagger,

I apologize for the delay in getting back to you. I've got in contact with our SDK team and this issue is currently part of their backlog and they are working on prioritisation. We will get back to you as soon as we'll make progress. Kind regards,

**Comment by stefaniacardenas** [comment]
Hi, this has been fixed in version `6.3.0`
